openapi-diff/src/core/OpenApiDiff/Program.cs

```diff
@@ -42,9 +44,19 @@ private static int Main(string[] args)
                 swaggerNew,
                 settings
             );
-            foreach (var msg in messages)
+
+            if (settings.JsonValidationMessages)
+            {
```

Comment:
is this for CLI usage ? and is the default false ?

Reply:
yeah, I need to remove completely. The flag is always true now.